Lumberjack.xcodeproj/project.pbxproj

```diff
@@ -1989,6 +1992,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
 				MODULEMAP_FILE = Framework/Lumberjack/CocoaLumberjack.modulemap;
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
AFAIK, Carthage pre-compiles your frameworks, so if the framework is precompiled in debug for 32-bit, probably won't be optimal for the 64-bit variation - meaning the active arch would be whatever machine compiled it on? Carthage is new to me, so I could be misspeaking.